plugins/modules/web_infrastructure/jira.py

```diff
@@ -81,18 +83,28 @@
     required: false
     description:
      - The comment text to add.
+     - Note that JIRA may not allow changing field values on specific transitions or states.
 
   status:
     type: str
     required: false
     description:
-     - The desired status; only relevant for the transition operation.
+     - Only used when C(operation) is I(transition), and a bit of aa misnomer, it actually refers to the transition name.
 
   assignee:
     type: str
     required: false
     description:
-     - Sets the assignee on create or transition operations. Note not all transitions will allow this.
+     - Sets the the assignee when C(operation) is I(create), I(transition) or I(edit).
+     - Recent versions of JIRA no longer accept an user name as an user identifier. In that case, use C(account_id) instead.
+     - Note that JIRA may not allow changing field values on specific transitions or states.
+
+  account_id:
+    type: str
+    description:
+     - Sets the account identifier for the assignee when C(operation) is I(create), I(transition) or I(edit).
```

Comment:
```suggestion
     - Sets the account identifier for the assignee when I(operation) is C(create), C(transition) or C(edit).
```
See https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-and-other-format-macros-within-module-documentation

Reply:
Looking forward to that semantic markup, so I no longer have to recall which one is which.